src/com/videojs/providers/HLSProvider.as

```diff
@@ -277,6 +278,37 @@ package com.videojs.providers{
           }
         }
 
+       /**
+         * Should return the list of audio-tracks that this content has.
+         */
+        public function get audioTracks():Array {
+            var _audioTracks:Array = [];
+
+            for each(var _audioTrackId:AudioTrack in _hls['audioTracks']) {
+              _audioTracks.push(_audioTrackId);
+            }
+
+            return _audioTracks;
+        }
+
+       /**
+         * Should return index of the currently selected audio track
+         */
+        public function get audioTrack():int {
+            return _hls['audioTrack'];
+        }
+
+        /**
+          * Should select an audio track based on its is index in audio-track list
+          */
+        public function set audioTrack(pValue:int):void {
+            _hls['audioTrack'] = pValue;
+
+            if (!isNaN(_position) && pValue != -1) {
```

Comment:
Is this necessary for this to work? I'd suggest removing this and seeing if things still work. I'm assuming this is from doing what is done for `set level`? I think that this wont be needed here because I don't think changing audio tracks has the same effect as manual switching a quality level.

Reply:
turns out this is needed